tests/test_schema.py

```diff
@@ -184,3 +184,12 @@ def test_schema_get_column_type(self):
 
         schema = MappingSchema({"foo": {"bar": parse_one("INT", into=exp.DataType)}})
         self.assertEqual(schema.get_column_type("foo", "bar").this, exp.DataType.Type.INT)
+
+    def test_schema_normalization(self):
+        schema = ensure_schema(
+            {"x": {"y": {"z": {"a": "INT", '"B"': "VARCHAR"}, "w": {"C": "INT"}}}}
+        )
+        expected_mapping = {"x": {"y": {"z": {"a": "INT", '"B"': "VARCHAR"}, "w": {"c": "INT"}}}}
```

Comment:
Will we have issues preserving the quotes of `"B"`? This is something we should be testing better...

Reply:
Again, good point. Will update this, thanks.

Reply:
I updated the test method -- happy to add more cases if you think I missed something.